Packs/CommonTypes/IndicatorFields/indicatorfield-CVSS_Score.json

```diff
@@ -17,12 +17,13 @@
     "content": true,
     "group": 2,
     "hidden": false,
+    "openEnded": false,
     "associatedTypes": [
-        "Infrastructure"
+        "CVE"
     ],
     "associatedToAll": false,
     "unmapped": false,
-    "unsearchable": false,
+    "unsearchable": true,
```

Comment:
Fields' "unsearchable" key cannot be modified, existing customers will keep previous behavior unless they explicitly uninstall and then install the pack from scratch

Reply:
It is known, an also true to all fields values.